### ISSUE TITLE
Cryptic errors when trying to build

### ISSUE THREAD

**Issue by talentlessguy** [osdevisnot/klap#19]
When running `klap build --pragma=h` I get this:

```
❯ yarn build
yarn run v1.19.2
warning package.json: No license field
$ klap build --pragma=h
klap@3.5.2 - Working on build...
(node:27361) UnhandledPromiseRejectionWarning: Error: Could not resolve entry module (src/money-target.js).
    at error (/home/v1rtl/Coding/money-target/node_modules/klap/dist/index.js:23:268012)
    at /home/v1rtl/Coding/money-target/node_modules/klap/dist/index.js:23:400303
    at async Promise.all (index 0)
    at async Promise.all (index 0)
    at async Promise.all (index 0)
```

no error message at all, so I can't figure out what's wrong.

here's the thing I try to build:

```jsx
import { h, render } from 'preact'

/** @jsx h */

render(<h1>Hello World</h1>, document.getElementById('app'))
```

I know that the error is that I didn't name the file properly but it should output this error in a more friendly way

**Comment by osdevisnot** [comment]
For this case, the `source` entry in `package.json` needs to point to the file you are trying to build. But I have to agree, an elaborated error message could have been more helpful.

**Comment by talentlessguy** [comment]
@osdevisnot I can try to PR this

**Comment by osdevisnot** [comment]
@talentlessguy the code here should be self explanatory for most part. Let me know if you face anything while working on this.

**Comment by talentlessguy** [comment]
@osdevisnot please check my pull request

**Comment by osdevisnot** [comment]
Fixed in v3.5.4
